FastEasyMapping/Source/Core/Mapping/FEMMapping.m

```diff
@@ -53,6 +53,25 @@ - (instancetype)initWithEntityName:(NSString *)entityName rootPath:(NSString *)r
     return self;
 }
 
+#pragma mark - NSCopying
+
+- (instancetype)copyWithZone:(NSZone *)zone {
+    FEMMapping *mapping = [[self.class allocWithZone:zone] initWithEntityName:self.entityName];
+    mapping.objectClass = self.objectClass;
```

Comment:
We also need to override setters and nil-out counterpart. Otherwise you can set both `objectClass` and `entityName` which might look strange 

```
- (void)setEntityName:(NSString *)entityName 
{
    _objectClass = nil;
    _entityName = [entityName copy];
}

- (void)setObjectClass:(Class)objetClass
{
    _objectClass = objetClass;
    _entityName = nil;
}
```